test/layer.spec.ts

```diff
@@ -153,20 +320,114 @@ describe("isGovCloud", () => {
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const datadogCdk = new Datadog(stack, "Datadog", {
+      nodeLayerVersion: NODE_LAYER_VERSION,
+      extensionLayerVersion: EXTENSION_LAYER_VERSION,
+      apiKey: "1234",
+      addLayers: true,
+      enableDDTracing: false,
+      flushMetricsToLogs: true,
+      site: "datadoghq.com",
+    });
+    datadogCdk.addLambdaFunctions([hello]);
+    expect(stack).toHaveResource("AWS::Lambda::Function", {
+      Layers: [
+        `arn:aws-us-gov:lambda:us-gov-east-1:${DD_GOV_ACCOUNT_ID}:layer:Datadog-Node10-x:${NODE_LAYER_VERSION}`,
+        `arn:aws-us-gov:lambda:us-gov-east-1:${DD_GOV_ACCOUNT_ID}:layer:Datadog-Extension:${EXTENSION_LAYER_VERSION}`,
       ],
     });
+  });
+
+  it("adds a GovCloud extension layer along with a GovCloud python layer ", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "us-gov-east-1",
+      },
+    });
+    const hello = new lambda.Function(stack, "HelloHandler", {
+      runtime: lambda.Runtime.PYTHON_3_7,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const datadogCdk = new Datadog(stack, "Datadog", {
+      pythonLayerVersion: PYTHON_LAYER_VERSION,
+      extensionLayerVersion: EXTENSION_LAYER_VERSION,
+      apiKey: "1234",
+      addLayers: true,
+      enableDDTracing: false,
+      flushMetricsToLogs: true,
+      site: "datadoghq.com",
+    });
+    datadogCdk.addLambdaFunctions([hello]);
     expect(stack).toHaveResource("AWS::Lambda::Function", {
       Layers: [
-        `arn:aws-us-gov:lambda:us-gov-east-1:${DD_GOV_ACCOUNT_ID}:layer:Datadog-Node10-x:${nodeLayerVersion}`,
+        `arn:aws-us-gov:lambda:us-gov-east-1:${DD_GOV_ACCOUNT_ID}:layer:Datadog-Python37:${PYTHON_LAYER_VERSION}`,
+        `arn:aws-us-gov:lambda:us-gov-east-1:${DD_GOV_ACCOUNT_ID}:layer:Datadog-Extension:${EXTENSION_LAYER_VERSION}`,
       ],
     });
   });
+
 });
+
+describe("generateLambdaLayerId", () => {
+  it("generates a lambda ID consisting of the prefix, runtime, and hash value delimited by hyphens", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "us-west-2",
+      },
+    });
+    const hello = new lambda.Function(stack, "NodeHandler", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromAsset("test"),
+      handler: "hello.handler",
+    });
+    const lambdaFunctionArn = "functionArn";
+    const runtime: string = hello.runtime.name;
+    const lambdaLayerId: string = generateLambdaLayerId(lambdaFunctionArn,runtime);
+    const lambdaLayerIdParts: string[] = lambdaLayerId.split("-");
+    expect(lambdaLayerIdParts[0]).toEqual("DatadogLayer");
+    expect(lambdaLayerIdParts[1]).toEqual("nodejs10.x");
+    expect(crypto.createHash("sha256").update(lambdaFunctionArn).digest("hex")).toEqual(lambdaLayerIdParts[2]);
```

Comment:
There very well could be a better way of testing this function along with the generateExtensionLayerId function but this is what came to mind.  I am open to suggestions.

Reply:
I think you could simplify this test a lot by just passing in strings.

`GenerateLambdaLayerId()` just takes in two strings and returns a new string. It doesn't really matter where those strings come from. So, all the stuff about creating a Stack with a Lambda function, then pulling the runtime out of it, doesn't really have anything directly to do with `GenerateLambdaLayerId()`.

I think it would be sufficient to have a really simple test here: pass in two strings and confirm that the result is what we would expect. Same for `generateExtensionLayerId`.